pyomo/core/tests/unit/test_sets.py

```diff
@@ -210,8 +210,8 @@ def test_addInvalid(self):
         # is, the default within is None
```

Comment:
It looks like `black` might have gone a bit crazy on this file. Can I please ask you to revert it and add back the lines you actually changed?